server.go

```diff
@@ -1124,13 +1126,13 @@ func chainUnaryInterceptors(interceptors []UnaryServerInterceptor) UnaryServerIn
 }
 
 func (s *Server) processUnaryRPC(t transport.ServerTransport, stream *transport.Stream, info *serviceInfo, md *MethodDesc, trInfo *traceInfo) (err error) {
-	sh := s.opts.statsHandler
-	if sh != nil || trInfo != nil || channelz.IsOn() {
+
```

Comment:
I believe we agreed you would add an `shs := s.opts.statsHandlers` here and use `shs` for the rest of the function; LMK if that's wrong.

Reply:
I updated the one in stream.go, missed this one. Updated.